app/assets/javascripts/pageflow/editor/views/edit_meta_data_view.js

```diff
@@ -34,9 +34,16 @@ pageflow.EditMetaDataView = Backbone.Marionette.Layout.extend({
       });
 
       this.input('credits', pageflow.TextAreaInputView);
-      this.input('author', pageflow.TextInputView);
-      this.input('publisher', pageflow.TextInputView);
-      this.input('keywords', pageflow.TextInputView);
+
+      this.input('author', pageflow.TextInputView, {
+        placeholder: pageflow.theming.attributes.default_author
+      });
+      this.input('publisher', pageflow.TextInputView, {
+        placeholder: pageflow.theming.attributes.default_publisher
+      });
+      this.input('keywords', pageflow.TextInputView, {
+        placeholder: pageflow.theming.attributes.default_keywords
+      });
```

Comment:
This goes back to our discussion about copying vs "inheriting" defaults. If we extend the fallback logic of the helper, I'm not sure whether things might get more confusing since there then can be

- entries with copied default meta data which do not change when the defaults in the theming change
- entries with emptied meta data attributes which then fallback to theming defaults but also change when the theming changes

Being able to retroactively change the meta data of a lot of entries by editing the theming seems like to much power to me. I'd say let's just display the defaults form the global config as placeholders here. Then if somebody chooses to use those by emptying his meta data fields, at least there is less risk of unexpected change because changing these defaults at least means editing the initializer and deploying. Makes sense?